tools/js-sdk-release-tools/src/autoGenerateInPipeline.ts

```diff
@@ -117,13 +144,15 @@ const optionDefinitions = [
     { name: 'sdkGenerationType', type: String },
     { name: 'inputJsonPath', type: String },
     { name: 'outputJsonPath', type: String },
+    { name: "apiVersion", type: String },
```

Comment:
please remove `apiversion` and `sdkReleaseType` since they can be specified in the file of `inputJsonPath`